base/commands/list/list_contains.go

```diff
@@ -0,0 +1,71 @@
+//go:build base || list
+
+package list
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/output"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/proto/codec"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/serialization"
+)
+
+type ListContainsCommand struct{}
+
+func (mc *ListContainsCommand) Init(cc plug.InitContext) error {
+	addValueTypeFlag(cc)
+	cc.SetPositionalArgCount(1, 1)
+	help := "Check if the value is present in the list."
+	cc.SetCommandHelp(help, help)
+	cc.SetCommandUsage("contains [value] [flags]")
+	return nil
+}
+
+func (mc *ListContainsCommand) Exec(ctx context.Context, ec plug.ExecContext) error {
+	listName := ec.Props().GetString(listFlagName)
+	ci, err := ec.ClientInternal(ctx)
+	if err != nil {
+		return err
+	}
+	// get the list just to ensure the corresponding proxy is created
+	if _, err := ec.Props().GetBlocking(listPropertyName); err != nil {
+		return err
+	}
```

Comment:
This code is not necessary since `clc list` also executes the same.

Reply:
I did not understand how `clc list` executes the same, could you explain?

Reply:
Sorry, I think I've confused it with something else.